core/src/com/goonsquad/galactictd/systems/initialization/ScoreScreenInitSystem.java

```diff
@@ -35,10 +38,28 @@ public ScoreScreenInitSystem(GalacticTDGame game) {
 
     @Override
     protected void populateWorld() {
+        createChangeScreenButton();
         createRedShip();
         createGreenShip();
     }
 
+    private int createChangeScreenButton() {
+        int changeScreenButton = archetypeBuilder.buildArchetype("invisible_button");
```

Comment:
> "invisible_button"
Probably should be put into a constant instead of using raw strings for this type of thing.